apm-sniffer/config/agent.config

```diff
@@ -57,6 +57,12 @@ agent.service_name=${SW_AGENT_NAME:Your_ApplicationName}
 # Snapshot transport to backend buffer size
 # profile.snapshot_transport_buffer_size=${SW_AGENT_PROFILE_SNAPSHOT_TRANSPORT_BUFFER_SIZE:50}
 
+# If true, skywalking agent will enable sending meters. Otherwise disable meter report.
+# meter.active=${SW_AGENT_METER_ACTIVE:true}
+
+# Report meters interval
+# meter.report_interval=${SW_AGENT_REPORT_INTERVAL:20}
```

Comment:
These are not widely used configurations, we should keep them in the document only.